specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -2231,6 +2384,23 @@
         }
       }
     },
+    "OutboundEnvironmentEndpoint": {
```

Comment:
This is a similar model to "InboundEnvironmentEndpoint" except it doesn't include the ports. Is this intentional?

Reply:
Yes, this is intentional. Inbound endpoints intentionally leave out the Fully Qualified Domain Name to avoid including infrastructure implementation details. Outbound endpoints include the FQDN so that they can be whitelisted by a layer 7 firewall. We don't have the same support for inbound endpoints so we only have the IP Address and port.